Sources/Moya/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -219,7 +219,7 @@ public extension NetworkLoggerPlugin.Configuration {
         // MARK: Defaults
 
         public static func defaultDataFormatter(_ data: Data) -> String {
-            return String(data: data, encoding: .utf8) ?? "## Cannot map data to String ##"
+            String(decoding: data, as: UTF8.self)
```

Comment:
Hmm, in this case I wonder what should be the default formatter. On one hand we want to print everything as a string, but probably we will get blobs of data from images or zips or other file formats that just do not make sense to print in the console... thoughts @LucianoPAlmeida @amaurydavid?

Reply:
I think that for blobs of data like that, the better option will be just `[Data: 15 bytes]`, and this is short and give reasonable information. And if the user wants to see the content is always possible to use a custom right

Reply:
I've never used this non optional decoding function and don't know its behavior. 
If it forces any data to be decoded into utf8 even if it's not from a String we indeed might end up with some gibberish. If keeping the optionnal decoding prevent that I say we keep it.

Reply:
I just reverted :)

Reply:
awesome, thanks @LucianoPAlmeida!